### DISCUSSION TITLE
Future of New and Noteworthy

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse/lsp4e#789] [General]
For the last few releases (since 0.20.2 about 18 months ago) there have been no edits to the changelog:

https://github.com/eclipse/lsp4e/blob/c23efae274bdc0a17586bfba4c8bba360e898301/CHANGELOG.md#L1-L6

What should we do going forward? Can we just add a link to the top of the changelog pointing at https://github.com/eclipse/lsp4e/releases and maybe updating https://github.com/eclipse/lsp4e/tree/master#changelog---new-and-noteworthy too?

If it sounds reasonable I will put together a PR.

**Comment by mickaelistria** [comment]
Hi,
The N&N was (and still can be) a communication tool to communicate the activity of the project and attract potential contributors. It seems like in the current state, we manage to get those contributors even without maintaining the N&N (most likely thanks to our reactivity on GitHub issues and PRs) and we do maintain some GitHub releases entries that can serve the purpose of hosting a changelig, so there is less urge to maintain the N&N.
So I agree we can stick to a link to changelog of releases; unless someone commits in maintaining it properly as a dedicated file in the future.

**Reply by jonahgraham** [reply]
Thanks @mickaelistria I will provide a PR for this. If in the future someone wants to add special changelog entries then they can just do that.
